pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -369,12 +371,21 @@ private void initializeDevice() {
     deviceState.system.software = new HashMap<>();
     deviceState.system.software.put("firmware", "v1");
     deviceState.system.last_config = new Date(0);
-    devicePoints.extraField = configuration.extraField;
 
-    if (configuration.extraPoint != null && !configuration.extraPoint.isEmpty()) {
-      addPoint(makePoint(configuration.extraPoint,
+    // Pubber runtime options
+    if (!configuration.options.extraField.isEmpty()) {
```

Comment:
should be "extraField != null" -- if somebody really wants to specify an empty extra field then they should be able to.